### PR TITLE
Fixes #3492

### LINKED ISSUE #3492: A very minor documentation issue
At https://github.com/sunpy/sunpy/blame/master/docs/guide/tour.rst#L52 it says that the timeseries data is random....but I don't think it is.

### PR DESCRIPTION
**Pull request by Raahul-Singh**
### Description

Fixes #3492

### CODE REVIEW

**Comment by Raahul-Singh** [comment]
@nabobalis Could you please tell me why the code coverage test is failing? :sweat_smile:

**Comment by nabobalis** [comment]
I don't know but It's safe to ignore.

File: changelog/3603.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Updated the docstring for TimeSeries in ~docs.guide.tour.rst describing the file used in the codeblock following the docstring.
```
Comment (nabobalis) [inline changelog/3603.doc.rst]:
```suggestion
Updated the tour guide to reflect that the time series is not random data. 
```

File: docs/guide/tour.rst [hunk 2]
```diff
@@ -48,8 +48,10 @@ TimeSeries
 SunPy handles time series data, fundamental to the study of any real world
 phenomenon, by creating a TimeSeries object. A timeseries consists of two parts;
 times and measurements taken at those times. The data can either be in your
-current Python session, alternatively within a local or remote file. Let's
-create some fake data and pass it into a timeseries object.
+current Python session, alternatively within a local or remote file.  In the code block that follows,
+data is taken from a file containing samples from GOES satellite's X-ray Sensors (XRS).
+
+Reference : `GOES XRS Homepage <https://www.swpc.noaa.gov/products/goes-x-ray-flux>`_
```
Comment (nabobalis) [inline docs/guide/tour.rst]:
```suggestion
```

File: docs/guide/tour.rst [hunk 3]
```diff
@@ -48,8 +48,10 @@ TimeSeries
 SunPy handles time series data, fundamental to the study of any real world
 phenomenon, by creating a TimeSeries object. A timeseries consists of two parts;
 times and measurements taken at those times. The data can either be in your
-current Python session, alternatively within a local or remote file. Let's
-create some fake data and pass it into a timeseries object.
+current Python session, alternatively within a local or remote file.  In the code block that follows,
```
Comment (nabobalis) [inline docs/guide/tour.rst]:
```suggestion
current Python session, alternatively within a local or remote file. 
In the code block that follows,
```

File: docs/guide/tour.rst [hunk 4]
```diff
@@ -48,8 +48,10 @@ TimeSeries
...
+data is taken from a file containing samples from GOES satellite's X-ray Sensors (XRS).
```
Comment (nabobalis) [inline docs/guide/tour.rst]:
```suggestion
from a file containing samples from the GOES satellite's X-ray Sensors (XRS).
```

**Comment by Raahul-Singh** [comment]
> I don't know but It's safe to ignore.

Cool. I'll read up on it just to be clear, thank you!

File: docs/guide/tour.rst [hunk 5]
```diff
@@ -48,8 +48,10 @@ TimeSeries
 SunPy handles time series data, fundamental to the study of any real world
 phenomenon, by creating a TimeSeries object. A timeseries consists of two parts;
 times and measurements taken at those times. The data can either be in your
-current Python session, alternatively within a local or remote file. Let's
-create some fake data and pass it into a timeseries object.
+current Python session, alternatively within a local or remote file. 
+In the code block that follows, data is taken from a file containing samples 
+from a file containing samples from the GOES satellite's X-ray Sensors (XRS).
+
```
Comment (nabobalis) [inline docs/guide/tour.rst]:
```suggestion
```

**Comment by nabobalis** [comment]
Thanks again @Raahul-Singh

**Comment by Raahul-Singh** [comment]
> Thanks again @Raahul-Singh

Thank you too for helping a noobie like me!
Looking forward towards contributing and learning more! :smile:

**Pull request merged**